spec/lib/os_places_api/client_spec.rb

```diff
@@ -204,33 +204,12 @@
         Postcode.create(postcode: postcode, results: [{ "DPA" => old_results }])
       end
 
-      it "should query OS Places API and update cached data when `update: true` is passed" do
+      it "should query OS Places API and update cached data" do
```

Comment:
This should now move to its own block at the end of the file, as this block is all contained within `describe "#locations_for_postcode" do` (line 4). We're describing a different method here.